go.mod

```diff
@@ -5,5 +5,6 @@ go 1.16
 require (
 	github.com/go-logr/logr v1.0.0
 	github.com/pkg/errors v0.9.1 // indirect
+	github.com/stretchr/testify v1.7.0
```

Comment:
any way we can NOT take this dep?

Reply:
`JSONeq` provides very nice error messages when an assertion fails. I might be possible to hand-craft the comparison, but it would be inferior compared to the code in `testify`. It also would be more work.

Is it really that bad to depend on it? In my experience it is fairly popular, well-maintained and stable.

Reply:
That's `gomega`. I agree, that one has issues. But IMHO `testify` is better.